src/app/pages/task-calendar/replication/replication-form/replication-form.component.ts

```diff
@@ -784,6 +786,18 @@ export class ReplicationFormComponent {
                 }
             }
         )
+
+        entityForm.formGroup.controls['speed_limit'].valueChanges.subscribe((value) => {
+            const speedLimitField = _.find(this.fieldConfig, {name: "speed_limit"});
+            const filteredValue = this.storageService.convertDataMeasurementStrings(value);
+                    if (isNaN(filteredValue)) {
+                        speedLimitField['hasErrors'] = true;
+                        speedLimitField['errors'] = 'Invalid value'
```

Comment:
If this error string is shown to the user, it should show an example of correct usage.  If formatting can be included in that message, the values should be in <i> tags or a monospace font or both.
```suggestion
                        speedLimitField['errors'] = 'Invalid value. Valid values are numbers followed by unit letters, like 256k or 1G.'
```